sort.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+function checkForNextLine($arr)
+{
+    $length = sizeof($arr);
+
+    for ($i = 0; $i < $length; $i++) {
+        if (strpos($arr[$i], "\n") === false) {
+            $arr[$i] .= "\n";
+        }
+    }
+
+    return $arr;
+}
+
+function bubbleSort($sort_array)
+{
+    $array_indexes = count($sort_array);
+    $sorted = false;
+
+    $checked = checkForNextLine($sort_array);
```

Comment:
Please also call it in the "main function", not in the bubbleSort function as it has nothing to do with bubble sort.

Reply:
Which main function? we don't have a main function.

Reply:
I mean the same place where you call the bubbleSort function.